google/cloud/internal/rest_retry_loop.h

```diff
@@ -121,8 +121,8 @@ auto RestRetryLoop(std::unique_ptr<RetryPolicy> retry_policy,
     -> google::cloud::internal::invoke_result_t<Functor, RestContext&,
                                                 Request const&> {
   return RestRetryLoop(std::move(retry_policy), std::move(backoff_policy),
-                       idempotency, std::forward<Functor>(functor), Options{},
-                       request, location);
+                       idempotency, std::forward<Functor>(functor),
+                       internal::CurrentOptions(), request, location);
```

Comment:
I assume the plan is to move this to a param in a follow up.

Reply:
Yes, see the #12359 for details.